src/stronghold.hs

```diff
@@ -309,12 +318,12 @@ main = do
       start (runStoreOpSnapSQL sql) (read portString)
     _ -> putStrLn "Expected stronghold [port] [zookeeper host string]"
 
-start :: (forall a. StoreOp a -> Snap a) -> Int -> IO ()
+start :: (forall a. SD.StoreOp a -> Snap a) -> Int -> IO ()
 start runStoreOp port = do
   let config =
         applyAll [
-          setPort port,
-          setAccessLog (writeTo stdout),
-          setErrorLog (writeTo stderr)
-        ] defaultConfig
-  simpleHttpServe (config :: Config Snap ()) (site runStoreOp)
+          Server.setPort port,
+          Server.setAccessLog (writeTo stdout),
+          Server.setErrorLog (writeTo stderr)
+        ] Server.defaultConfig
```

Comment:
I think it might be more clear to separate this out into

```
let
  configOpts =  [
    Server.setPort port,
    Server.setAccessLog (writeTo stdout),
    Server.setErrorLog (writeTo stderr)
  ]
  config = applyAll configOpts Server.defaultConfig
```

Only do this if you agree.